nuxeo-runtime/nuxeo-launcher-commons/src/test/java/org/nuxeo/launcher/config/AbstractConfigurationTest.java

```diff
@@ -40,6 +42,8 @@
 
     protected File nuxeoHome;
 
+    protected static final Map<String, String> originSystemProps = new HashMap<>();
```

Comment:
- `static` because there's only one instance that we use for tests—it's a singleton, like the real system properties.
- `final` because the reference of this map should not be changed. It guarantees that the map will not be modified accidentally by any usages. Please notice that `final` does NOT mean the map is unmodifiable—it ensure the same reference of the object, but it's still usable by using methods like `Map#put`

Reply:
If you keep it `static final`, please rename the variable `ORIGIN_SYSTEM_PROPS`

Reply:
It's used by non-static methods and the test class instance is already implicitly a singleton. You must not use `final` here this would be confusing.